pkg/clients/release/plans.go

```diff
@@ -44,13 +44,13 @@ func (r *ReleaseController) CreateReleasePlan(name, namespace, application, targ
 }
 
 // CreateReleasePlanAdmission creates a new ReleasePlanAdmission using the given parameters.
-func (r *ReleaseController) CreateReleasePlanAdmission(name, namespace, environment, origin, policy, serviceAccountName string, applications []string, autoRelease bool, pipelineRef *tektonutils.PipelineRef, data *runtime.RawExtension) (*releaseApi.ReleasePlanAdmission, error) {
+func (r *ReleaseController) CreateReleasePlanAdmission(name, namespace, environment, origin, policy, serviceAccountName string, applications []string, blockReleases bool, pipelineRef *tektonutils.PipelineRef, data *runtime.RawExtension) (*releaseApi.ReleasePlanAdmission, error) {
 	releasePlanAdmission := &releaseApi.ReleasePlanAdmission{
 		ObjectMeta: metav1.ObjectMeta{
 			Name:      name,
 			Namespace: namespace,
 			Labels: map[string]string{
-				releaseMetadata.AutoReleaseLabel: strconv.FormatBool(autoRelease),
+				"releases.appstudio.openshift.io/block-releases": strconv.FormatBool(blockReleases), // TODO - replace with imported constant once release-service go module updated
```

Comment:
Maybe I'm misunderstanding, but won't this PR be merged only after the release-service is updated for this anyway and thus this can be changed here before merging? Am I missing something? Or is that how this TODO is meant - to be updated before this PR is merged?

Reply:
The operator is already updated in development and staging. The new constant exists. The issue is that updating the release-service module version in this repo is go dependency hell